<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>260</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>480</height>
         </size>
        </property>
        <property name="title">
         <string>操作</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn_openInp">
           <property name="text">
            <string>打开inp文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_inpNodeNum">
           <property name="text">
            <string>inp节点数量：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_inpCellType">
           <property name="text">
            <string>inp网格类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_inpCellNodeNum">
           <property name="text">
            <string>单个inp网格节点数量：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_inpCellNum">
           <property name="text">
            <string>inp网格数量：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>选择Vtp网格类型</string>
            </property>
           </widget>
           <widget class="QComboBox" name="combo_CellType">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>UnstructuredGrid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PolyData</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_renderModel">
           <property name="text">
            <string>转化并渲染vtp模型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_addFieldData">
           <property name="text">
            <string>添加场数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_addPointData">
           <property name="text">
            <string>添加点数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QPushButton" name="btn_saveVtp">
            <property name="text">
             <string>保存VTP文件</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_cancel">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>230</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QVTKOpenGLWidget" name="vtk_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvtkopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
